### PR TITLE
update sketch -> figma on homepage

### PR DESCRIPTION
**Pull request by cowills**

### CODE REVIEW

File: docs/.vuepress/theme/Home.vue [hunk 1]
```diff
@@ -111,8 +111,8 @@
                     tag="h3"
                     modifier="heading-sans-400"
                     class="cdr-mb-space-one-x"
-                  >Cedar Sketch UI toolkit</cdr-text>
-                  <cdr-link :href="$withBase('/getting-started/as-a-designer/')" modifier="standalone">Download the Sketch library</cdr-link>
+                  >Cedar Figma UI toolkit</cdr-text>
```
Comment (mking18) [inline docs/.vuepress/theme/Home.vue:114]: Cedar Design Libraries

File: docs/.vuepress/theme/Home.vue [hunk 2]
```diff
@@ -111,8 +111,8 @@
...
+                  <cdr-link :href="$withBase('/getting-started/as-a-designer/')" modifier="standalone">Download the Figma library</cdr-link>
```
Comment (mking18) [inline docs/.vuepress/theme/Home.vue:115]: Download the toolkit

Comment (mking18) [inline docs/.vuepress/theme/Home.vue:115]: /getting-started/as-a-designer/#cedar-ui-toolkit

**Comment by cowills** [comment]
@mking18 if you click edit file you can make all those changes directly in the PR

**Pull request merged**